### PR TITLE
fix: update for latest framework and pieces

### PR DESCRIPTION
**Pull request by favna**
> [!IMPORTANT]
> Rebase & Merge

### CODE REVIEW

**Comment by favna** [comment]
Basically we need to be able to remove all the `ts-expect-error` comments without getting build errors.

The cause appears to be that module augmentations arent being loaded

Purely experimental but I've already tried setting type to module in each package.json but to no avail

**Review by vladfrangu — DISMISSED** [review]

**Comment by favna** [comment]
@sapphiredev pack

**Comment by favna** [comment]
@sapphiredev pack

**Comment by favna** [comment]
@sapphiredev pack

File: packages/logger/src/register.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
+import './index';
```
Comment (vladfrangu) [inline packages/logger/src/register.ts:1]: this isn't needed

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**